hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1778,7 +1778,7 @@
               digest.
             '''
       milestone: V2
-      tests: ["chip_sw_hmac_idle"]
+      tests: ["chip_sw_hmac_enc_idle"]
```

Comment:
There is still another instance of `chip_sw_hmac_idle` in this testplan. Is that supposed to be changed to `chip_sw_hmac_enc_idle`, too?

Reply:
Yes, sorry I didn't see it.

Reply:
Ah sorry I believe I have been looking at an outdated nightlies dashboard - nevermind!